### DISCUSSION TITLE
feat(online):  还要过滤几个本地地址,显示为localhost

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#3726] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

这两个地址在我这里会出现
::1%0
::1%250

**Comment by bb-auto** [comment]
@densen2014 Thank you for contacting us. We will give feedback later.

**Reply by ArgoZhang** [reply]
看起来这是 IPv6 地址，我们有一个方法可以转化为 IPv4 在翻译成 localhost

```
public static string ToIPv4String(this IPAddress? address)
{
    var ipv4Address = (address ?? IPAddress.IPv6Loopback).ToString();
    return ipv4Address.StartsWith("::ffff:") ? (address ?? IPAddress.IPv6Loopback).MapToIPv4().ToString() : ipv4Address;
}
```

**Comment by densen2014** [comment]
localhost	127.0.0.1 
ip6-localhost ip6-loopback	::1	 
ip6-localnet	fe00::0	 
ip6-mcastprefix	ff00::0	 
ip6-allnodes	ff02::1	 
ip6-allrouters	ff02::2	 
ip6-allhosts	ff02::3
